[IMP] Appraisals: Update main doc (#13141)
odoo/documentation: content/applications/hr/appraisals.rst

:nosearch:

===========

> Attendances

Author: I kept switching those names in my head!!! OMG! And as for the points- no problem! I put 3 originally when I thought I was doing a pretty good rewrite, and this new change wasn't decided upon till later. I absolutely agree!!

Regular appraisals are good for both the employees and the company, since they can accurately

> measure performance based on company goals, and show employees where they need to improve.

Reviewer: I'd like to take some time to provide a totally optional rewrite to the introduction. The original has all the key features and delivers the value proposition of the Appraisal app. It's already strong!

I've drafted a more concise version and walk you through **what changed** and **why I'm changing** it, so you understand my reasoning, and hopefully this guides you to be consistently concise in your documentation :blush: 

### Proposed rewrite
> The **Appraisals** app provides a structured, repeatable way to review performance and set goals. Managers schedule reviews, employees complete self-assessments, and both sides record feedback in one place. Custom templates, rating scales, and 360° feedback keep each review aligned with company objectives and highlight concrete next steps—information managers can later use for raises, promotions, or training plans.

### Why this version works better

| Goal | Original | Rewrite | Rationale |
|------|----------|---------|-----------|
| **Lead with value** | Opens by stating the app “is used to evaluate employee performance.” | Opens with the outcome: “a structured, repeatable way to review performance and set goals.” | Hooks readers with what they gain first, not the mechanics. |
| **Keep it concise** | Several sentences repeat why appraisals are useful. | Consolidates benefits into two focused sentences. | Reduces reading time while preserving meaning. |
| **Avoid generic phrasing** | “Good for both the employees and the company.” | Lists specific results: feedback, goals, raises, promotions, training plans. | Concrete examples feel more actionable than generic praise. |

Your draft contained all the right ingredients—I just repackaged them so readers can grasp the value faster. Feel free to use or adapt this rewrite, and let me know if any part of the reasoning isn’t clear. Happy to chat or pair on edits.

Great work getting the substance down; these small tweaks push it over the finish line. 🚀

The :guilabel:`Configuration` menu in the **Appraisals** application is where various settings are

> configured, frequencies can be set, appraisal templates can be edited, evaluation scales can be

Reviewer 2: ```suggestion
configured, frequencies are set, appraisal templates are edited, evaluation scales are
```

Reviewer 2: To match with the first item on this list

> managed, data for 360 feedback can be stored, skills can be managed, and goal tags can be created.

Reviewer 2: ```suggestion
managed, data for 360 feedback is stored, skills are managed, and goal tags are created.
```

appraisals are only created once a year (every twelve months).

To modify this schedule, change the number of months in the blank fields under the
:guilabel:`Appraisals Plans` section.

To have appraisals automatically scheduled *and* confirmed according to the schedule, tick the
checkbox next to :guilabel:`Appraisals Automation`.

.. important::
   If the :guilabel:`Appraisals Plans` section is modified, **all** empty :guilabel:`Next Appraisal
   Dates` are modified for **all** employees.

Enable the :guilabel:`360 Feedback` option to allow managers to request feedback from other
employees using a different survey form, at any time, independent of the appraisal schedule.

Typically, managers ask for feedback from other people who work with an employee they manage. This
includes the employee's various managers, coworkers, and direct reports.

By default, the preconfigured :guilabel:`360 Feedback` populates the :guilabel:`Default Template`
field.

To view the :guilabel:`360 Feedback` survey, click the :icon:`oi-arrow-right` :guilabel:`(Internal
link)` icon at the end of the :guilabel:`Default Template` field. The :guilabel:`360 Feedback`

> survey loads, and any desired changes to the survey can be made.

Reviewer 2: ```suggestion
survey loads, and any desired changes to the survey are then made.
```
